### ISSUE TITLE
chronosBindAverage.scopePrevious5SecondPeriod stat doesn't match the average of chronosBindInstantaneousCount

### ISSUE THREAD

**Issue by rkd-msw** [Metaswitch/chronos#329]
#### Symptoms

On a system with no activity, I repeatedly checked the value of chronosBindInstantaneousCount (which didn't change), then checked the value of chronosBindAverage.scopePrevious5SecondPeriod - which was several times higher.

```
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:44 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:45 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:46 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:47 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:47 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:48 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:49 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:49 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount
Wed  4 Jan 13:00:50 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 7
root@abra:~# date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindAverage  Wed  4 Jan 13:00:52 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5SecondPeriod = Gauge32: 54
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopeCurrent5MinutePeriod = Gauge32: 21
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5MinutePeriod = Gauge32: 21
```

#### Impact

Users will get a misleading view of the usage of their deployment depending on what statistic they query.

#### Release and environment 

`10.0.0-161216-~255.00.0-161216.020317`

#### Steps to reproduce

I don't know what gets it into this state.

**Comment by rkd-msw** [comment]
Diags packages from both Sprout nodes at `\\ad\csd\bigfiles\clearwater\issue-attachments\chronos\328`.

**Comment by johadalin** [comment]
I'm fairly sure this will be the same root cause as #328 and #330 , so it should be resolved. Passing back for re-test

**Comment by rkd-msw** [comment]
Fixed on 10.0.0-170118-~255.00.0:

```
rkd@abra:~/salinas$ date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindAverage
Sat 21 Jan 10:47:59 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5SecondPeriod = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopeCurrent5MinutePeriod = Gauge32: 74963
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5MinutePeriod = Gauge32: 62585
rkd@abra:~/salinas$ date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-2.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindAverage
Sat 21 Jan 10:48:01 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5SecondPeriod = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopeCurrent5MinutePeriod = Gauge32: 74963
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5MinutePeriod = Gauge32: 62585
rkd@abra:~/salinas$ date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-1.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-1.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindAverage
Sat 21 Jan 10:48:13 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5SecondPeriod = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopeCurrent5MinutePeriod = Gauge32: 74964
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5MinutePeriod = Gauge32: 62585
rkd@abra:~/salinas$ date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-1.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-1.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindAverage
Sat 21 Jan 10:48:14 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5SecondPeriod = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopeCurrent5MinutePeriod = Gauge32: 74964
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5MinutePeriod = Gauge32: 62585
rkd@abra:~/salinas$ date; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-1.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount; snmpwalk -m PROJECT-CLEARWATER-MIB -v2c -c clearwater stress-sprout-1.stress.clearwater.test PROJECT-CLEARWATER-MIB::chronosBindAverage
Sat 21 Jan 10:48:15 GMT 2017
PROJECT-CLEARWATER-MIB::chronosBindInstantaneousCount.0 = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5SecondPeriod = Gauge32: 74969
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopeCurrent5MinutePeriod = Gauge32: 74964
PROJECT-CLEARWATER-MIB::chronosBindAverage.scopePrevious5MinutePeriod = Gauge32: 62585
```
